### DISCUSSION TITLE
Space Cloud itself on serverless platforms (AWS Lambdas)

### DISCUSSION THREAD

**Discussion by iBobik** [spaceuptech/space-cloud#1584] [Ideas]
Hi,
I am interested in Space Cloud gateway on true serverless platform, so I does not have to care about Kubernetes, just pay Lambdas execution and some database for state (DynamoDB for configs?).

I see this is not documented use-case, so I ask for brainstorming it that could be possible.

**Comment by YourTechBud** [comment]
I'm not sure if it will work. Can tell me a bit more about your use case?

**Reply by iBobik** [reply]
I want to use Space Cloud gateway to access external services, another Lambdas or AWS resources. Could work well, I just does not need to run gateway nonstop and sometimes want to scale it horizontally, so running SC gateway in a Lambda would be ideal.

**Reply by YourTechBud** [reply]
It should be possible in theory. Let me checkout the feasibility for the same once.

Have you considered using fargate?

**Comment by iBobik** [comment]
Yes, considered, but Fargate could not scale to zero.


Honza Pobořil


22. 11. 2021 14:13:56, Noorain Panjwani ***@***.***> napsal:
